odoo/modules/loading.py

```diff
@@ -276,7 +281,10 @@ def load_test(idref, mode):
         if package.name is not None:
             registry._init_modules.add(package.name)
 
-    _logger.log(25, "%s modules loaded in %.2fs, %s queries", len(graph), time.time() - t0, odoo.sql_db.sql_counter - t0_sql)
+        _logger.info("Module %s loaded in %.2fs, %s (+%s) queries", module_name, time.time() - module_t0, cr.sql_log_count - init_module_query_count, odoo.sql_db.sql_counter - module_t0_sql)
```

Comment:
The log message is everything but clear.
Does the first query count include the tests?
And the message does not say what the parenthesized value is.

I suggest something more explicit like
```
Module %s loaded in %.2fs, %s queries + %s queries for tests
```